examples/flows/standard/conditional-flow-for-if-else/README.md

```diff
@@ -2,18 +2,18 @@
 
 This example is a conditional flow for if-else scenario.
 
+By following this example, you will learn how to create a conditional flow using the `activate config`.
+
+## Flow description
+
 In this flow, it checks if an input query passes content safety check. If it's denied, we'll return a default response; otherwise, we'll call LLM to get a response and then summarize the final results.
 
 The following are two execution situations of this flow:
-- if input query passes content safety check:
-  ![content_safety_check_passed](content_safety_check_passed.png)
-- else:
-  ![content_safety_check_failed](content_safety_check_failed.png)
+- if input query passes content safety check: ![content_safety_check_passed](content_safety_check_passed.png)
+- else: ![content_safety_check_failed](content_safety_check_failed.png)
```

Comment:
The preview looks bad: 
![Uploading image.png…]()

Reply:
Updated.